storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/operations/put/PreparedPutObjectTest.java

```diff
@@ -263,8 +266,51 @@ public void shouldThrowExceptionIfNoTypeMappingWasFoundWithoutAffectingDbComplet
                     "db was not affected by this operation, please add type mapping for this type");
 
             verify(storIOSQLite).lowLevel();
+            verify(storIOSQLite).defaultScheduler();
             verify(internal).typeMapping(Object.class);
             verifyNoMoreInteractions(storIOSQLite, internal);
         }
     }
-}
+
+    public static class OtherTests {
```

Comment:
Because they also check `PreparedPutoObject` operation but don't related together as all tests without type mapping or type mapping errors verifications.